{"name": "test_d_3", "status": "failed", "statusDetails": {"message": "AssertionError: assert (3 + 1) == 3", "trace": "self = <DemoPyTest.b_folder.test_d.TestD object at 0x03783250>\n\n    def test_d_3(self):\n>       assert 3 + 1 == 3\nE       assert (3 + 1) == 3\n\nb_folder\\test_d.py:24: AssertionError"}, "start": 1592389159475, "stop": 1592389159476, "uuid": "a3d87c87-5fad-4030-afa1-9361921f8a9b", "historyId": "8cfe35eb3f4e0ef8d4a56e642ce4de3a", "testCaseId": "2e001e950f352bf0a8c0abb01ffcb209", "fullName": "b_folder.test_d.TestD#test_d_3", "labels": [{"name": "parentSuite", "value": "b_folder"}, {"name": "suite", "value": "test_d"}, {"name": "subSuite", "value": "TestD"}, {"name": "host", "value": "R10459"}, {"name": "thread", "value": "23444-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "b_folder.test_d"}]}